mycroft/skills/skill_updater.py

```diff
@@ -13,28 +13,60 @@
 # limitations under the License.
 #
 """Periodically run by skill manager to update skills and post the manifest."""
+import json
 import os
-import sys
-from datetime import datetime
-from time import time
+from os.path import join, isfile
 import xdg.BaseDirectory
+from json_database import JsonStorage
 
 from mycroft.api import DeviceApi, is_paired
 from mycroft.configuration import Configuration, BASE_FOLDER
-from mycroft.configuration.ovos import is_using_xdg
-from mycroft.util import connected
-from combo_lock import ComboLock
 from mycroft.util.log import LOG
-from mycroft.skills.msm_wrapper import build_msm_config, create_msm, MsmException
-from mycroft.util.file_utils import get_temp_path
+from mycroft.skills.skill_loader import get_skill_directories
 
-ONE_HOUR = 3600
-FIVE_MINUTES = 300  # number of seconds in a minute
 
+class SeleneDeviceSkills(JsonStorage):
+    def __init__(self, api=None):
```

Comment:
Docstring methods and what this class is for

Reply:
This is a simple persistent dict, basically functionality extracted from msm. It is only meant for internal usage. The point is to allow uploading of skills manifest to selene even without msm